docs/core/porting/cppcli.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Tools for porting to .NET Core
+description: Learn about porting C++/CLI projects to .NET Core
+author: mjrousos
+ms.author: 
+ms.date: 12/12/2019
+---
+
+# How to port a C++/CLI project to .NET Core
+
+Beginning with .NET Core 3.1 and Visual Studio 2019 16.4, C++/CLI projects can be updated to target .NET Core. This allows Windows desktop applications which may use C++/CLI projects as an interop layer between managed and native code to be ported to .NET Core. This article describes how to port C++/CLI projects from .NET Framework to .NET Core 3.1.
+
+## C++/CLI .NET Core limitations
+
+There are some important limitations to porting C++/CLI projects to .NET Core compared to C# or VB projects:
```

Comment:
I'd drop the `C# or VB` projects part. You'll get some F# developers tweeting at you for ignoring them 🙂 

But more seriously, since the following points have nothing to do with a language, it's best to just say they're different from "typical" .NET Core projects.